objects.md

```diff
@@ -205,12 +216,15 @@ of `Instant` and the second is a `string` representing a valid IANA timezone.
 
 #### ZonedDateTime.prototype
 
-`ZonedDateTime.prototype` includes all properties and methods of `OffsetDateTime.prototype`, plus the following additions and replacements.
+`ZonedDateTime.prototype` includes all properties and methods of `OffsetDateTime.prototype` except `plus` & `minus`.
```

Comment:
Doesn't that destroy the whole idea of a [class hierarchy](https://github.com/tc39/proposal-temporal/issues/120#user-content-class-relationships)? Why should ZonedDateTime not support the same arithmetic as OffsetDateTime?